patient-portal/src/main/java/org/isf/patientportal/model/hospitalevent/HospitalEvent.java

```diff
@@ -108,6 +111,10 @@ public HospitalEvent updateWith(HospitalEvent item) {
 			this.date = item.date;
 			this.patient = item.patient;
 			this.eventType = item.eventType;
+<<<<<<< PP-11-Fix_errors_api_update
+			this.readDate = item.readDate;
+=======
+>>>>>>> develop
```

Comment:
there a conflict here to be solved (choose between PP-11 version or develop one)